### DISCUSSION TITLE
Sunbird Saas | Incorrect Telemetry Data in QUML Player Affecting Exam Results

### DISCUSSION THREAD

**Discussion by Pritams43** [Sunbird-Ed/Community#736] [Bugs]
**Issue :** We have identified a critical issue with the QUML player where incorrect telemetry data is being recorded after the completion of exams. Specifically, it has come to our attention that some of the answers are being interchanged with other question DO IDs in the telemetry JSON data. This means that a user's answer to one question is being incorrectly logged under a different question.

**Details of the Issue:**

**Problem**: Incorrect mapping of answers to questions in telemetry data.

**Impact:** Inaccurate representation of user responses and exam results.

**Affected Data:** Both the database and Meta-Base reports show inconsistencies where answers are misaligned with questions

**Implications:**

1.User Trust: This issue could make users lose trust in our system because they can't rely on its accuracy. This could lead to dissatisfaction and damage our reputation.

2.Assessment Integrity: The exam results do not reflect the true performance of the users, compromising the integrity of our assessments.

3.Decision-Making: Wrong data can lead to poor decisions based on inaccurate reports, affecting educational and organizational goals.

4.Resource Allocation: Resources and time might be wasted due to wrong data, leading to inefficiency. Additionally, They are arranging refresher training for those who failed due to incorrect results leads to a loss of manpower and money.

5.Compliance Risks: For institutions that need accurate data for legal or regulatory reasons, this issue could cause serious problems. Furthermore, since Prerak status is changed based on the results, there is a possibility that an under performing Prerak is promoted while a deserving Prerak is not.

This issue has been observed in the EG project use case. It is possible that similar problems exist in other areas as well, though they have not yet surfaced.

@mohitga @vinukumar-vs

**Comment by HarishGangula** [comment]
@Pritams43  Can ask this in https://github.com/orgs/Sunbird-inQuiry/discussions ?

**Comment by HarishGangula** [comment]
As we haven't received a response from you, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
